[TM] Chp3-1 More inputs to the vector attribute data section (#3299)
qgis/QGIS-Documentation: source/docs/training_manual/basic_map/vector_data.rst

===============================================================================

Vector data is arguably the most common kind of data you will find in the daily

> use of GIS. It describes geographic data in terms of points, lines and polygons on a

Reviewer: ```suggestion
use of GIS. The 2D vector model describes the location and shape of geographic features using points, lines and polygons.
```

Reviewer 2: ```suggestion
use of GIS. Vector data defines geographic features as points, lines and polygons on a
```

Reviewer: @tomchadwin, I am not sure why this is an improvement. Could you explain your reason for suggesting this change?

Reviewer: What about "The 2D vector model" and "location and shape".

Reviewer 2: The reasons I suggested removing both of those are:

- I think the use of 2D could be confusing, given how many geometry types now supported in QGIS have a Z attribute
- I think the word "model" is unnecessary, and could potentially be confused with other meanings (such as "models" in 3D software)
- vector datasets don't only define location and shape, but also attributes

Reviewer 2: > points, lines and polygons (plus attributes) are very seldom exact representation of geographical objects. They are more or less accurate abstractions

A good point. How about in my suggestion changing "define" to "represent"? That allows for any levels of precision/abstraction, but doesn't limit it to geometry (as opposed to attribute data).

Reviewer: Thank for the very valuable input! Let us see what @DelazJ thinks about this...

> coordinate plane. It is usually used to store discrete features, like roads and

Reviewer: ```suggestion
It is usually used to store discrete features, like roads and
```

> city blocks, Every object in a vector dataset is called a **feature**,

Reviewer: ```suggestion
city blocks. The objects in a vector dataset are called **features**,
```

> and is associated with data that describes that feature.

Author: I guess this end of sentence might also need update following @havatv reformulation

> and is associated with data that describes that feature.

Reviewer: ```suggestion
and are associated with data that describe their location and properties.
```

Reviewer 2: ```suggestion
and contain data that describe their location and properties.
```

Reviewer: Fine.

represent **where** objects are in space, but also tells you **what** those
objects are.

From the previous exercise, you should have the :guilabel:`rivers` layer

> loaded in your map. The lines that you can see right now is merely the

Reviewer 2: ```suggestion
loaded in your map. The lines that you can see right now are merely the
```

> position of the rivers; this is the *spatial data*.

Reviewer 2: ```suggestion
position of the rivers: this is the *spatial data*.
```

To see all the data available to you, with the :guilabel:`rivers` layer
selected in the Layers panel, click on this button: |openTable|

It will show you a table with more data about the :guilabel:`rivers` layer.
This extra dialog is the :guilabel:`Attribute table`. Each row, called

> *record*, represents a river *feature* and each column, also called *field*,

Reviewer 2: I like @havatv's suggestions, but would revert to "each" from the original.

> *record*, represents a river *feature* and each column, also called *field*,

Reviewer: For consistency I have updated the next line instead.

> represents a property of the river. Cells show *attributes*.

Reviewer: ```suggestion
and represents a property of the river. Cells show *attributes*.
```
